packages/mdc-chips/chip-set/index.js

```diff
@@ -0,0 +1,65 @@
+/**
+ * @license
+ * Copyright 2016 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import MDCComponent from '@material/base/component';
+
+import MDCChipSetAdapter from './adapter';
+import MDCChipSetFoundation from './foundation';
+import {MDCChip} from '../chip';
+
+/**
+ * @extends {MDCComponent<!MDCChipSetFoundation>}
+ * @final
+ */
+class MDCChipSet extends MDCComponent {
+  /**
+   * @param {...?} args
+   */
+  constructor(...args) {
+    super(...args);
+    /** @private {!Array<!Element>} */
+    this.chips;
+  }
+
+  /**
+   * @param {!Element} root
+   * @return {!MDCChipSet}
+   */
+  static attachTo(root) {
+    return new MDCChipSet(root);
+  }
+
+  initialize(chipFactory = (el) => new MDCChip(el)) {
+    this.chips = this.instantiateChips_(chipFactory);
+  }
+
+  /**
+   * @return {!MDCChipSetFoundation}
+   */
+  getDefaultFoundation() {
+    return new MDCChipSetFoundation(/** @type {!MDCChipSetAdapter} */ (Object.assign({
+      hasClass: (className) => this.root_.classList.contains(className),
+    })));
+  }
+
+  instantiateChips_(chipFactory) {
```

Comment:
Does this need type information to compile with closure?

Also, this PR doesn't seem to be adding `mdc-chips` to `closureWhitelist` in package.json... should we be doing that from the outset with this package?